src/Leaderboard.js

```diff
@@ -9,7 +9,7 @@ class Leaderboard {
     const board = document.getElementById('board');
     const row = document.createElement('div');
     row.className = 'score';
-    row.innerHTML = `<p class="w-100 d-flex justify-content-between p-2 m-0"> ${Score.user}: ${Score.score} </p>`;
+    row.innerHTML = `<p class="w-100 d-flex justify-content-between py-2 px-5 m-0"> <span class="user">${Score.user}:</span> ${Score.score} </p>`;
```

Comment:
[OPTIONAL] It would be good if you validate inputs to only accept numbers. You could also add a valid range of numbers, entirely up to you.

![image](https://user-images.githubusercontent.com/79114233/142544728-0dd94a73-6c06-42bf-8261-31b3bfb0af04.png)

Reply:
Done!